src/Project.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Sandboxes\Api;
+
+class Project
+{
+    private string $id;
+    private string $mlflowUri;
+
+    private string $createdTimestamp;
+    private string $updatedTimestamp;
+
+    public static function fromArray(array $in): self
+    {
+        $project = new Project();
+        $project->setId((string) $in['id']);
+        $project->setMLflowUri($in['mlflowUri'] ?? '');
+        $project->setCreatedTimestamp($in['createdTimestamp']);
+        $project->setUpdatedTimestamp($in['updatedTimestamp'] ?? '');
+
+        return $project;
+    }
+
+    public function toArray(): array
+    {
+        $result = [];
+        if (!empty($this->id)) {
+            $result['id'] = $this->id;
+        }
+        if (!empty($this->mlflowUri)) {
+            $result['mlflowUri'] = $this->mlflowUri;
+        }
+
+        if (!empty($this->createdTimestamp)) {
+            $result['createdTimestamp'] = $this->createdTimestamp;
+        }
+        if (!empty($this->updatedTimestamp)) {
+            $result['updatedTimestamp'] = $this->updatedTimestamp;
+        }
+
+        return $result;
+    }
+
+    public function toApiRequest(): array
+    {
+        $array = $this->toArray();
+        unset($array['id']);
+        unset($array['createdTimestamp']);
+        unset($array['updatedTimestamp']);
```

Comment:
this is a little funny looking but i get it :-) I think it's ok

Reply:
Yes, but it's copied from the Sandbox class, there it is the same. 🙂